.allow_skipping_tests

```diff
@@ -60,6 +60,7 @@ notifications/base_notification.rb
 notifications/followup_notification.rb
 notifications/followup_resolved_notification.rb
 notifications/youth_birthday_notification.rb
+notifications/delivery_methods/sms.rb
```

Comment:
ideally let's not *add* to the skipped tests file

Reply:
For sure. I'm beginning to figure out how this could be tested, which will hopefully lay the groundwork for testing the other notifications in .allow_skipping_tests